meta-lmp-base/recipes-kernel/linux/kmeta-linux-lmp-5.15.y.inc

```diff
@@ -1,4 +1,4 @@
-KERNEL_META_REPO ?= "git://github.com/foundriesio/lmp-kernel-cache.git"
+KERNEL_META_REPO ?= "git://github.com/angolini/lmp-kernel-cache.git"
 KERNEL_META_REPO_PROTOCOL ?= "https"
-KERNEL_META_BRANCH ?= "linux-v5.15.y"
-KERNEL_META_COMMIT ?= "d1d97152bd5e02683e417b62d4868fb2f85bbe6d"
+KERNEL_META_BRANCH ?= "mx93"
```

Comment:
Before merging, don't forget to replace this commit using the regular fio branch instead of yours.

Reply:
I'm working on this right now